src/libYARP_sig/src/Image.cpp

```diff
@@ -126,7 +126,7 @@ class ImageStorage {
     }
 
     void resize(int x, int y, int pixel_type,
-                int pixel_size, int quantum, bool topIsLow);
+                int quantum, bool topIsLow);
```

Comment:
I'm afraid that this function is largely used in the downstream project( `icub-main` for sure). We should leave the old signature deprecated and add this one.

Reply:
Forget it, I thought was `Image::resize` :sweat_smile:

Reply:
I think this method is hidden in the c++ file, so it cannot be used outside